cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,62 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
+describe('Article flow', () => {
+  let user;
+  let article;
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+      cy.login(user.email, user.username, user.password);
+      cy.getCookie('auth').should('exist');
+      cy.reload();
+      cy.task('generateArticle').then((generatedArticle) => {
+        article = generatedArticle;
```

Comment:
There is a potential issue here: the article is generated asynchronously after the page reload. Ensure that the article is fully generated before proceeding with the tests that depend on it. Consider using `cy.wrap` or another method to handle this asynchronous operation properly.